dhis2/src/main/java/org/motechproject/dhis2/service/impl/SettingsServiceImpl.java

```diff
@@ -60,6 +61,12 @@ private synchronized void loadSettings() {
             LOGGER.debug("Loading DHIS2 settings...");
             Gson gson = new Gson();
             settings = gson.fromJson(jsonText, Settings.class);
+            URI uri = new URI(settings.getServerURI());
```

Comment:
@kmadej I've added URI normalization for removal of multiple slashes, but it leaves trailing slash, so I still remove it manually.